### ISSUE TITLE
Driver is missing requirement adafruit-blinka-displayio

### ISSUE THREAD

**Issue by kiler129** [adafruit/Adafruit_CircuitPython_DisplayIO_SH1107#19]
I followed instructions on https://docs.circuitpython.org/projects/displayio-sh1107/en/latest/index.html verbatim and I've got:

```
root@rpi4-usb:~# mkdir project-name && cd project-name
python3 -m venv .venv
source .venv/bin/activate
pip3 install adafruit-circuitpython-displayio-sh1107
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting adafruit-circuitpython-displayio-sh1107
  Using cached https://www.piwheels.org/simple/adafruit-circuitpython-displayio-sh1107/adafruit_circuitpython_displayio_sh1107-1.6.1-py3-none-any.whl (6.5 kB)
Collecting Adafruit-Blinka
  Using cached https://www.piwheels.org/simple/adafruit-blinka/Adafruit_Blinka-8.32.0-py3-none-any.whl (324 kB)
Collecting Adafruit-PlatformDetect>=3.53.0
  Using cached https://www.piwheels.org/simple/adafruit-platformdetect/Adafruit_PlatformDetect-3.60.0-py3-none-any.whl (23 kB)
Collecting Adafruit-PureIO>=1.1.7
  Using cached https://www.piwheels.org/simple/adafruit-pureio/Adafruit_PureIO-1.1.11-py3-none-any.whl (10 kB)
Collecting pyftdi>=0.40.0
  Using cached https://www.piwheels.org/simple/pyftdi/pyftdi-0.55.0-py3-none-any.whl (145 kB)
Collecting adafruit-circuitpython-typing
  Using cached https://www.piwheels.org/simple/adafruit-circuitpython-typing/adafruit_circuitpython_typing-1.10.2-py3-none-any.whl (11 kB)
Collecting RPi.GPIO
  Using cached RPi.GPIO-0.7.1.tar.gz (29 kB)
  Preparing metadata (setup.py) ... done
Collecting rpi-ws281x>=4.0.0
  Using cached rpi_ws281x-5.0.0.tar.gz (64 kB)
  Preparing metadata (setup.py) ... done
Collecting sysv-ipc>=1.1.0
  Using cached sysv_ipc-1.1.0.tar.gz (99 kB)
  Preparing metadata (setup.py) ... done
Collecting pyusb!=1.2.0,>=1.0.0
  Using cached https://www.piwheels.org/simple/pyusb/pyusb-1.2.1-py3-none-any.whl (58 kB)
Collecting pyserial>=3.0
  Using cached https://www.piwheels.org/simple/pyserial/pyserial-3.5-py2.py3-none-any.whl (90 kB)
Collecting adafruit-circuitpython-busdevice
  Using cached https://www.piwheels.org/simple/adafruit-circuitpython-busdevice/adafruit_circuitpython_busdevice-5.2.6-py3-none-any.whl (7.5 kB)
Collecting adafruit-circuitpython-requests
  Using cached https://www.piwheels.org/simple/adafruit-circuitpython-requests/adafruit_circuitpython_requests-2.0.5-py3-none-any.whl (11 kB)
Collecting typing-extensions~=4.0
  Using cached https://www.piwheels.org/simple/typing-extensions/typing_extensions-4.9.0-py3-none-any.whl (32 kB)
Installing collected packages: sysv-ipc, RPi.GPIO, pyserial, Adafruit-PlatformDetect, typing-extensions, rpi-ws281x, pyusb, Adafruit-PureIO, pyftdi, adafruit-circuitpython-requests, adafruit-circuitpython-busdevice, adafruit-circuitpython-typing, Adafruit-Blinka, adafruit-circuitpython-displayio-sh1107
  DEPRECATION: sysv-ipc is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for sysv-ipc ... done
  DEPRECATION: RPi.GPIO is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for RPi.GPIO ... done
  DEPRECATION: rpi-ws281x is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for rpi-ws281x ... done
Successfully installed Adafruit-Blinka-8.32.0 Adafruit-PlatformDetect-3.60.0 Adafruit-PureIO-1.1.11 RPi.GPIO-0.7.1 adafruit-circuitpython-busdevice-5.2.6 adafruit-circuitpython-displayio-sh1107-1.6.1 adafruit-circuitpython-requests-2.0.5 adafruit-circuitpython-typing-1.10.2 pyftdi-0.55.0 pyserial-3.5 pyusb-1.2.1 rpi-ws281x-5.0.0 sysv-ipc-1.1.0 typing-extensions-4.9.0
(.venv) root@rpi4-usb:~/project-name# cat > test.py
//code from example

(.venv) root@rpi4-usb:~/project-name# python test.py
Traceback (most recent call last):
  File "/root/project-name/test.py", line 2, in <module>
    import displayio
ModuleNotFoundError: No module named 'displayio'
```

Are there some steps missing from the instruction?

**Comment by makermelissa** [comment]
It looks like you're setting up for the Blinka environment. Try running `pip3 install adafruit-blinka-displayio`.

**Comment by makermelissa** [comment]
Looks like it's missing the dependency in the requirements, so I changed the title.

**Comment by FoamyGuy** [comment]
This was resolved in #21. Blinka displayio is now listed in requirements.txt
